Версия 0.(0)1

iamolegga opened the pull request:
Пока что версия недоделанная, но хоть какая-то :)
Генерируем пока что статью и 1 комментарий.
Через пару дней приведу к конечному варианту. Пожалуйста, пока что не
комментируйте, так как это  – далеко не конечный вариант :)

Commit:
Версия 0.(0)1

Пока что версия недоделанная, но хоть какая-то :)
Генерируем пока что статью и 1 комментарий.
Через пару дней приведу к конечному варианту. Пожалуйста, пока что не
комментируйте, так как это  – далеко не конечный вариант :)

azproduction commented on repin/js/scripts_generator.js:
```diff
@@ -0,0 +1,318 @@
+        $(this).hide();
+    });
+
+    // ===================================================== Функции ==================================================
```
Лучше создать новый модуль, чемделить код таким образом.

azproduction commented on repin/js/scripts_generator.js:
```diff
@@ -0,0 +1,318 @@
+     */
+
+    function findPrefInText(whereFrom, lengthOfPrefix) {
+        var wordsInText = new Array(),
```
`new Array() -> []`

azproduction commented on repin/js/scripts_generator.js:
```diff
@@ -0,0 +1,318 @@
+
+    function findPrefInText(whereFrom, lengthOfPrefix) {
+        var wordsInText = new Array(),
+            wordsInText = $(whereFrom).text().match(/\S+/g),
```
Нужно кэшировать `$(whereFrom).text()`

azproduction commented on repin/js/scripts_generator.js:
```diff
@@ -0,0 +1,318 @@
+            }
+            else {
+                var finalSuf = findSuf;
+            };
```
Зачем Statement терминировать с `;`?

azproduction commented on repin/js/scripts_generator.js:
```diff
@@ -0,0 +1,318 @@
+     */
+
+    function addSuf(where, addStr) {
+        $(where).text(
```
Обычно пишут

``` js
$(where).text(function () {

});
```

azproduction commented on repin/js/scripts_generator.js:
```diff
@@ -0,0 +1,318 @@
+     *
+     */
+
+    var kM = -1;
```
`kM` ?

azproduction commented on repin/js/scripts_generator.js:
```diff
@@ -0,0 +1,318 @@
+     * Добавляем заготовку для нового текста и секцию комментариев
+     */
+
+    $('.container').append('<article class="new_text"></article>');
```
Для подобных случаев лучше использовать шаблонизацию

azproduction commented on repin/js/scripts_generator.js:
```diff
@@ -0,0 +1,318 @@
+     * найденного в тексте префикса
+     */
+    
+    while ($('.new_text').text().match(/\S+/g).length < NEWARTICLESIZE) {
```
Нужно `$('.new_text').text()` "закэшировать"

azproduction commented on repin/js/scripts_generator.js:
```diff
@@ -0,0 +1,318 @@
+        + 'a></li><li><a href="#">Civilization 2: десять лет без перезапуск'
+        + 'а 13.06.2012</a></li><li><a href="#">Steam для Linux. Скоро 05.0'
+        + '6.2012</a></li><li><a href="#">Steam взломан 11.11.2011</a></li>'
+        + '</ul></div></div>');
```
Это явно наблон

azproduction commented:
Это приложение можно где-нибудь посмотреть на github.io?

Commit:
новая версия

вставляются теги, картинки, комментарии, подключены настройки, есть 1
баг:
иногда (редко) комментарии вставляются во внутрь статьи с пустым
текстом))

важное замечание: теги со всеми словами которые написаны между
открывающим и закрывающим тегом считаются как одно слово

в будущем, если будет интересно, сделаю так чтобы и внутри тегов слова
тоже менялись
